addons/calendar/models/calendar_event.py

```diff
@@ -48,6 +48,8 @@
     ('custom', 'Custom')
 ]
 
+INSTANT_MAIL_CONFIRMATION_MAX_N_ATTENDEES = 20
```

Comment:
Would probably make an ir config parameter for this, as it would allow easy customization if you have a strong mail server, without having to do custom code. Something like ``calendar.meeting_mail_send_limit`` or something like that (don't define it in data, just use 20 as default value if it does not exist)